test_create_new_repository/Step1.swift

```diff
@@ -9,8 +9,62 @@
 import SwiftUI
 
 struct Step1: View {
+    @State private var title: String = ""
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack(alignment: .leading, spacing: 0) {
+            VStack(alignment: .leading, spacing: 0) {
+                HStack(alignment: .center, spacing: 0) {
+                    VStack(alignment: .leading, spacing: 0) {
+                        Text("스낵수정")
+                            .font(.system(size: 24))
+                        HStack(alignment: .center, spacing: 0) {
+                            Text("1단계 ")
+                                .font(.system(size: 24))
+                                .foregroundColor(.blue)
+                            Text("(1/5)")
+                                .font(.system(size: 24))
+                        }
+                    }.padding(.bottom, 4)
+                        .frame(width: 118, height: 72, alignment: .leading)
+                    Spacer()
+                }
+                Text("스낵 타이틀은 “[지역] 할 일”로 구체적으로 적으면 참여율이 올라가겠죠?!")
+                    .font(.system(size: 14))
+                    .foregroundColor(.gray)
+                    .padding(.bottom, 24)
+            }
+            VStack(alignment: .leading, spacing: 0) {
+                HStack(alignment: .center, spacing: 0) {
+                    Text("스낵 타이틀")
+                        .font(.system(size: 12))
+                    Spacer()
+                    Text("0/30")
```

Comment:
실제 값이 카운트 되려면, 다음처럼 구현할 수 있습니다.
```suggestion
                    Text("\(self.title.count)/30")
```

Reply:
반영했습니다.